goose-migration/docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gcr.io/cloudsql-docker/gce-proxy as cloudsql-proxy
+
+FROM golang:1.19-alpine
+
+# Copy cloud-proxy binary
+COPY --from=cloudsql-proxy /cloud_sql_proxy /cloud_sql_proxy
+
+RUN go install github.com/pressly/goose/v3/cmd/goose@latest
```

Comment:
Do we want to grab the latest or lock it to a specific version?

Reply:
@furrowb don't see a reason for not grabbing latest considering they don't release too often, could throw it in as an arg if you think it's best?

Reply:
@furrowb updated, you can now input that